playbooks/roles/install-epel/tasks/main.yml

```diff
@@ -1,11 +1,4 @@
 ---
 
-- name: Install Extra Packages for Enterprise Linux (EPEL)
-  yum:
-    name:
-      - "https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
```

Comment:
Def better to have these sections out of main.yml. Would be nice if could declare paths etc. as k/v dict fields. What is py best practice?